examples/multimodal/configs/video_agg.yaml

```
@@ -0,0 +1,43 @@
+    resources:
+      gpu: 1
+  common-configs: [model, block-size, max-model-len]
+
+VllmEncodeWorker:
+  tensor-parallel-size: 1
+  router: random
+  ServiceArgs:
+    workers: 1
+    resources:
+      gpu: 1
+  common-configs: [model]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)